mohawk/util.py

```diff
@@ -268,3 +267,37 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
```

Comment:
I think it would be nicer to define this like `get_bewit(resource, expires_in=30)` and then make the adjustment down below as `exp=resource.timestamp + expires_in`.